plugin/al/plugin/AL_USDMayaTestPlugin/AL/usdmaya/commands/test_ProxyShapeSelect.cpp

```diff
@@ -27,8 +29,45 @@
 #include <maya/MSelectionList.h>
 #include <maya/MStringArray.h>
 
+#if defined(WANT_UFE_BUILD)
+#include <ufe/globalSelection.h>
+#include <ufe/observableSelection.h>
+#include <ufe/path.h>
+#endif
+
 using AL::maya::test::buildTempPath;
 
+namespace {
+
+bool isAlive(AL::usdmaya::nodes::ProxyShape* proxy, SdfPath path)
```

Comment:
Not sure I understand the difference between isAlive() and isSelected() --- both seem to look in the UFE selection for an SdfPath?

Reply:
As far as I can tell, the distinction between the two is that `isAlive()` will also look for children of the specified path, i.e.
```cpp
return selection->contains(ufePath) || selection->containsDescendant(ufePath);
```
I have updated the method to use UFE selection instead of the string search anyhow.